elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -54,14 +56,16 @@
             .put(RSQLOperators.GREATER_THAN, Operator.GT)
             .put(RSQLOperators.GREATER_THAN_OR_EQUAL, Operator.GE)
             .put(RSQLOperators.LESS_THAN_OR_EQUAL, Operator.LE)
+            .put(isNullOp, Operator.ISNULL)
             .build();
 
     private final RSQLParser parser;
     private final EntityDictionary dictionary;
 
     public RSQLFilterDialect(EntityDictionary dictionary, Set<ComparisonOperator> rsqlOperators) {
         // Match "filter[<type>]"
-
+        //Adding "=isnull=" operator
+        rsqlOperators.add(isNullOp);
```

Comment:
I don't think this constructor works as intended.  We should probably remove it.